load/InputFileset.py

```diff
@@ -138,16 +148,32 @@ def toString(self):
 	## This method is used instead of _setFilenames when a specific list of files has been provided
 	def _downloadSelectedFiles(self, fileList):
 		results = []
-		s3Client = AWSSession.shared().s3Client
+		session = boto3.Session(profile_name = self.config.s3_aws_profile)
+		s3Client = session.client('s3')
 		directory = self.config.directory_upload_aws + self.filesetId
+
```

Comment:
I don't understand what is going on here, and why we need to be changing anything related to OS directories. Please explain

Reply:
@bradflood It is because e.g. into the folder: `~/upload/ABIWBTN_ET-usx/Abidji_N2ABIWBT_USX/` we have a sub directory with the filesetPath (`Abidji_N2ABIWBT_USX`), so we need to make that we can go through sub-directory when it starts to download the files.

Reply:
I suspect the changes to directory structure (to move content up a directory), will mean this is not necessary. Let's continue to discuss

Reply:
@bradflood, you are right, It is no longer necessary, I have removed this new lines and I have tested it and I got the same result.

Reply:
@bradflood I have updated these lines, could you check it? The new lines are: https://github.com/faithcomesbyhearing/dbp-etl/pull/6/files#r825125137

Reply:
@bradflood It has been removed.